src/azure-cli/azure/cli/command_modules/synapse/manual/operations/kustopool.py

```diff
@@ -147,3 +149,78 @@ def synapse_kusto_pool_detach_follower_database(client,
                        kusto_pool_name=kusto_pool_name,
                        resource_group_name=resource_group_name,
                        follower_database_to_remove=follower_database_to_remove)
+
+
+def synapse_kusto_script_show(cmd,
+                              workspace_name,
+                              script_name):
+    client = cf_kusto_script(cmd.cli_ctx, workspace_name)
+    return client.get_by_name(kql_script_name=script_name)
+
+
+def synapse_kusto_script_create(cmd,
+                                resource_group_name,
+                                workspace_name,
+                                script_name,
+                                definition_file,
+                                kusto_pool_name=None,
+                                kusto_database_name=None,
+                                no_wait=False):
+
+    client = cf_kusto_script(cmd.cli_ctx, workspace_name)
+    query = read_file_content(definition_file)
+    metadata = KqlScriptContentMetadata(language="kql")
+    current_connection = KqlScriptContentCurrentConnection(pool_name=kusto_pool_name,
+                                                           database_name=kusto_database_name)
+    script_content = KqlScriptContent(query=query, metadata=metadata, current_connection=current_connection)
+    properties = KqlScript(content=script_content)
+    kql_script = KqlScriptResource(name=script_name, properties=properties)
+    return sdk_no_wait(no_wait,
+                       client.begin_create_or_update,
+                       kql_script_name=script_name,
+                       kql_script=kql_script
+                       )
+
+
+def synapse_kusto_script_delete(cmd,
+                                workspace_name,
+                                script_name,
+                                no_wait=False):
+    client = cf_kusto_script(cmd.cli_ctx, workspace_name)
+
+    return sdk_no_wait(no_wait,
+                       client.begin_delete_by_name,
+                       kql_script_name=script_name)
+
+
+def synapse_kusto_script_list(cmd,
+                              workspace_name):
+    client = cf_kusto_scripts(cmd.cli_ctx, workspace_name)
+    return client.get_all()
+
+
+def synapse_kusto_script_export(cmd, workspace_name, output_folder, script_name=None):
+    if script_name is not None:
+        kusto_script_client = cf_kusto_script(cmd.cli_ctx, workspace_name)
+        kql_script = kusto_script_client.get_by_name(script_name)
+        path = os.path.join(output_folder, script_name + '.kql')
+        print(kql_script.properties.as_dict())
```

Comment:
Is this `print` for debug? If yes, please delete this line, if no, better to use `logger.warning`.

Reply:
print for debug, also following previous code: https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/synapse/manual/operations/artifacts.py#L271

Reply:
I don't think previous code is elegant😂 We shouldn't keep debug code in released cli